shell/probe/raindrop.ps1

```diff
@@ -30,6 +29,7 @@ function Execute {
         $stdout = if ($stdout) { [string]::Join("; ", $stdout) } else { "" }
         $stderr = Get-Content -Path $stderrTempFile
         $stderr = if ($stderr) { [string]::Join("; ", $stderr) } else { "" }
+        if ($timeoutVar) {$stdErr = [string]::Join(" ", $stderr, "TEST TIMEOUT; ") }
```

Comment:
`stdErr` doesn't look like it is used anywhere and the rest of the logs are `;` separated. How about this
```suggestion
        if ($timeoutVar) {$stderr += "; TEST TIMEOUT") }
```

Reply:
or alternatively this and include $timeoutMsg in the `Write-Host` on line 34.
```suggestion
        $timeoutMsg = if ($timeoutVar) { "TEST TIMEOUT" } else { "" }
```

Reply:
I think I like the approach I took. Thoughts?

Reply:
looks good to me